Resources/Textures/Objects/Misc/qm_clipboard.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Clipboard sprites are by Endecc (Github) for SS14 Wizard's Den, taken from ***NO LINK YET, ME***, qm_clipboard_paper is a modified version of paper from bureaucracy.rsi.",
+  "copyright": "Clipboard sprites are by Endecc (Github) for SS14 Wizard's Den, taken from https://github.com/space-wizards/space-station-14/pull/21023, qm_clipboard_paper is a modified version of paper from bureaucracy.rsi.",
```

Comment:
just say its for SS14 dont need to link to your own pr :trollface:

Reply:
I probably misunderstood the practice of just linking external PRs, but it probably doesn't hurt to have there.
But I am clueless, I can take it out if necessary to merge

Reply:
im not maint :trollface: but ive never seen anyone else do it

Reply:
I think I've done it for every other one of my sprite-adding PRs without issue, unless I'm gaslighting myself
We will see what a maint thinks of it if it's a problem